apps/system/js/remote/simple_app_window_manager.js

```diff
@@ -0,0 +1,77 @@
+/* global BaseModule */
+'use strict';
+
+(function(exports) {
```

Comment:
nit: you don't need to redefine window to exports here because base module export it for you.